src/masterless.py

```diff
@@ -0,0 +1,60 @@
+from fabric.api import local, task
+from decorators import requires_project, requires_aws_stack, echo_output
+from buildercore import bootstrap, config, core
+from buildercore.utils import ensure
+import logging
+from functools import wraps
+LOG = logging.getLogger(__name__)
+
+def requires_master_server_access(fn):
+    @wraps(fn)
+    def wrapper(*args, **kwargs):
+        import master
+        ensure(master.server_access(), "this command requires access to the master server. you don't have it.")
+        return fn(*args, **kwargs)
+    return wrapper
+
+@task
+@requires_project
+@requires_master_server_access
+def launch(pname, instance_id=None):
+    import cfn
+    cfn.launch(pname, instance_id, 'masterless')
+    # opportunity to do post-launch things here
```

Comment:
exactly, the `masterless` `aws-alt` config and the `masterless` `ec2` options are two different things, so I would like to name them with two different names